### PR TITLE
Styling signup & login pages

### PR DESCRIPTION
**Pull request by damienaries**
### What this PR does (required):
- Adapt styling for login and signup forms and pages to match mocks

### Screenshots / Videos (required):
- Login
 Screen Shot 2021-08-17 at 2 17 39 PM
- Signup
Screen Shot 2021-08-17 at 2 11 10 PM

### Any information needed to test this feature (required):
- Please let me know if element positioning and sizes look off in other browsers/environments

### CODE REVIEW

File: client/src/pages/Login/Login.tsx [hunk 1]
```diff
@@ -51,6 +50,7 @@ export default function Login(): JSX.Element {
               </Grid>
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
+            <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
```
Comment (rajivtitus) [inline client/src/pages/Login/Login.tsx:53]: Can we rename this component for increase semantics? Currently it only renders a button and calling it AuthHeader is misleading.

File: client/src/pages/SignUp/SignUp.tsx [hunk 2]
```diff
@@ -40,18 +40,18 @@ export default function Register(): JSX.Element {
...
             <Grid container>
               <Grid item xs>
                 <Typography className={classes.welcome} component="h1" variant="h5">
-                  Create an account
+                  Sign up
                 </Typography>
               </Grid>
             </Grid>
             <SignUpForm handleSubmit={handleSubmit} />
+            <AuthHeader linkTo="/login" asideText="Already a member?" btnText="Login" />
```
Comment (rajivtitus) [inline client/src/pages/SignUp/SignUp.tsx:54]: Same as comment above

**Review by rajivtitus — APPROVED** [review]
Good work!

Just a minor comment, please address before merging. Thanks!

**Comment by damienaries** [comment]
Done, merging

**Pull request merged**